libaugrim/src/two_phase_commit/coordinator_algorithm.rs

```diff
@@ -29,6 +29,13 @@ use super::CoordinatorState;
 use super::TwoPhaseCommitContext;
 use super::TwoPhaseCommitMessage;
 
+// The timeout for receiving all decision acks before continuing to the next epoch.
```

Comment:
typo in commit message: coordinaotr -> coordinator

Reply:
fixed